plugin-packs/postcss-preset-env/test/basic.ff66.expect.css

```diff
@@ -181,7 +173,7 @@
 }
 
 .test-hwb-function {
-	background-color: rgba(0, 195, 255, .5);
+	background-color: rgba(0, 196, 255, 0.5);
```

Comment:
Small differences like these are fine.
A single point increase/decrease should be fine in the context of a fallback.

Any color transform from a cylindrical to a rectangular color space will be sensitive to implementation details and we completely overhauled ours.